sentry-apache-http-client-5/src/test/kotlin/io/sentry/transport/apache/ApacheHttpClientTransportTest.kt

```diff
@@ -100,4 +102,22 @@ class ApacheHttpClientTransportTest {
         sut.close()
         verify(fixture.client).close(CloseMode.GRACEFUL)
     }
+
+    @Test
+    fun `flush waits till all requests are finished`() {
```

Comment:
Ideally another test that verifies it'll bail after timeout even if all events didnt' get flushed